src/Components/CurrWeatherCard/CurrWeatherCard.jsx

```diff
@@ -0,0 +1,49 @@
+import styles from './CurrWeatherCard.module.scss';
+import endpoints from '../../Helpers/endpoints';
+
+export default function CurrWeatherCard(props) {
+  let locationData = props.locationData;
+  let currWeather = props.currWeather;
```

Comment:
props can be destructured so no need to create new variables

Reply:
done